spring-integration-xml/src/main/resources/org/springframework/integration/xml/config/spring-integration-xml-4.2.xsd

```diff
@@ -749,6 +749,30 @@
 							<xsd:union memberTypes="xsd:boolean xsd:string" />
 						</xsd:simpleType>
 					</xsd:attribute>
+					<xsd:attribute name="output-properties">
+						<xsd:annotation>
+							<xsd:documentation>
+								A set of output properties that will be used to override any of the same properties
+								in affect for the transformation.
+							</xsd:documentation>
+							<xsd:appinfo>
+								<tool:annotation kind="ref">
+									<tool:expected-type type="java.util.Properties"/>
+								</tool:annotation>
+							</xsd:appinfo>
+						</xsd:annotation>
+					</xsd:attribute>
+					<xsd:attribute name="iterator" default="true">
+						<xsd:annotation>
+							<xsd:documentation>
+								The iterator mode: 'true' (default) to return an 'java.util.Iterator'
+								for splitting 'payload', 'false to return a 'java.util.List'.
```

Comment:
We should explain the benefit - the list contains transformed nodes whereas with the iterator each node is transformed while iterating.